plugins/flutter_aepedgebridge/ios/flutter_aepedgeBridge.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'flutter_aepedgebridge'
-  s.version          = '4.0.0'
+  s.version          = '4.0.2'
```

Comment:
podspec update from 4.0.0 to 4.0.2 due to I miss to update the 4.0.1 version

Reply:
Will this be an issue? Or it is fine if they are not able to access 4.0.1 version?

Reply:
This version shows up when doing the pod update.

e.g flutter_aepcore (4.0.2)

So should match to Pubspec version.  It doesn't affect functionality.

<img width="365" alt="podupdate" src="https://github.com/adobe/aepsdk_flutter/assets/29806547/873b6206-3b01-402e-9023-0d12f525d6e3">